telegram/ext/jobqueue.py

```diff
@@ -229,7 +284,8 @@ def run_daily(self, callback, time, days=Days.EVERY_DAY, context=None, name=None
                   tzinfo=time.tzinfo,
                   context=context,
                   name=name,
-                  job_queue=self)
+                  job_queue=self,
+                  is_monthly=None)
```

Comment:
This is not needed here, since `is_monthly` defaults to `None`. Please remove.

Reply:
done